frontend/src/pages/ApartmentPage.tsx

```diff
@@ -98,13 +100,21 @@ const ApartmentPage = (): ReactElement => {
 
   type Fields = keyof typeof reviewData[0];
 
-  const showConfirmationToast = () => {
-    setShowConfirmation(true);
+  const showToast = (setState: (value: React.SetStateAction<boolean>) => void) => {
```

Comment:
Ok that this stuff is repeated between `ApartmentPage.tsx` and `LandlordPage.tsx` for now. There are no more common updates between the two pages, and soon `LandlordPage.tsx` will look very different. A lot of it will be changed, and all of the currently repeated stuff will not be applicable to the page anymore.